database.yaml

```diff
@@ -2,6 +2,58 @@
 - name: Setup database
   hosts: database
   tasks:
+    - name: Update repositories 
+      become: true
+      apt:
+        update_cache: yes
+        cache_valid_time: 86400 #A day
+        
+    - name: install mysql server
+      become: true
+      apt:
+        name: 
+          - mysql-server
+          - python3-pymysql  
+        state: present
+
+        
+# Set MySQL root Password on Ansible/Ubuntu - https://linuxhint.com/set_mysql_root_password_ansible/ 
+    - name: Enable empty password for MySQL root user
+      become: true
+      shell: mysql -u root -e 'UPDATE mysql.user SET plugin="mysql_native_password"
+       WHERE user="root" AND host="localhost"; FLUSH PRIVILEGES;'
+
+    - name: Create SQL user 
+      mysql_user:
+        login_user: 'root'
+        login_password: ''
+        name: wp_user
+        host: '%'
+        password: wp_password
+        priv: "*.*:ALL"
+        state: present
+     
+    - name: Create DB for wordpress
+      mysql_db:
+        login_user: 'root'
+        login_password: ''
+        name: wp_db
+        state: present
+    
+    - name: Open MySQL port
+      become: true
+      ufw:
```

Comment:
Using/not using ufw is not consistent. I would recommend to use one tool, or another. It would remove some mess

Reply:
I understand. I prefer UFW as its simpler and has a nice ansible integration. However, I couldn't get my NFS server to work by using ufw. I even tried opening the nfs one by one, but it still didn't work. Therefore, I restrained the use of firewalld exclusively for this playbook.